assets/install_ellisys.sh

```diff
@@ -1,36 +1,44 @@
 #!/bin/bash
 
-MSI_PATH="/assets/tmp/bta.msi"    # Store outside docker-image
+BTA_PATH="/assets/tmp/bta.msi"    # Store outside docker-image
+BTA_URL="https://www.ellisys.com/better_analysis/bta_get.php"
+#BTA_URL="https://www.ellisys.com/better_analysis/bta_get.php?v=5.0.8393.22590"
+EBQ_VIEWER_PATH="/assets/tmp/EBQ_viewer.exe"    # Store outside docker-image
 INSTALL_FOLDER="C:\Ellisys"
+BTA_INSTALL_FOLDER="${INSTALL_FOLDER}\Ellisys Bluetooth Analyzer"
+EBQ_VIEWER_INSTALL_FOLDER="${INSTALL_FOLDER}\Ellisys Bluetooth Qualifier Viewer"
 
-test -e "${MSI_PATH}" || {
+BTA_SIZE=$(wc -c $BTA_PATH)
+BTA_DL_SIZE=$(wget $BTA_URL --spider --server-response -O - 2>&1 | sed -ne '/Content-Length/{s/.*: //;p}')
+if ( test -e "${BTA_PATH}" && [ "${BTA_SIZE%% *}" = "$BTA_DL_SIZE" ])
+then
+    echo "### Already have the latest installer, skipping download"
+else
     echo "### Could not find Ellisys installer. Downloading..."
-    wget -O "${MSI_PATH}" "https://www.ellisys.com/better_analysis/bta_get.php"
-#   wget -O "${MSI_PATH}" "https://www.ellisys.com/better_analysis/bta_get.php?v=5.0.7690.40428"
-}
+    wget -O "${BTA_PATH}" "${BTA_URL}"
+fi
```

Comment:
Magic :-)  I'd go for something simpler: If the file exists locally, use it. If not, download.